examples/VDM.SAPObjectNodeType-sample.xml

```diff
@@ -0,0 +1,105 @@
+
+      <EntityType Name="A_Country">
+        <Key>
+          <PropertyRef Name="Country" />
+        </Key>
+        <Property Name="Country" Type="Edm.String" />
+        <Property Name="CountryThreeLetterISOCode" Type="Edm.String" MaxLength="3" />
+      </EntityType>
+
+      <Annotations Target="this.A_Country">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Country" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <Annotations Target="this.A_Country/Country">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Country" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <EntityType Name="A_Region">
+        <Key>
+          <PropertyRef Name="Country" />
+          <PropertyRef Name="Region" />
+        </Key>
+        <Property Name="Country" Type="Edm.String" />
+        <Property Name="Region" Type="Edm.String" />
+        <Property Name="CountryThreeLetterISOCode" Type="Edm.String" />
+      </EntityType>
+
+      <Annotations Target="this.A_Region">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Region" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <Annotations Target="this.A_Region/Country">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Country" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <Annotations Target="this.A_Region/Region">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Region" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+
+      <EntityType Name="A_AnyEntity">
+        <Key>
+          <PropertyRef Name="AnyEntity" />
+        </Key>
+        <Property Name="AnyEntity" Type="Edm.String" Nullable="false" />
+        <Property Name="Country" Type="Edm.String" />
+        <Property Name="Region" Type="Edm.String" />
+        <Property Name="CountryThreeLetterISOCode" Type="Edm.String" MaxLength="3" />
+      </EntityType>
+
+      <Annotations Target="this.A_AnyEntity/Country">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Country" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <Annotations Target="this.A_AnyEntity/Region">
+        <Annotation Term="VDM.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" String="Region" />
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <Annotations Target="this.A_AnyEntity/CountryThreeLetterISOCode">
```

Comment:
Is there a deeper reason why the analogous property in the other entity types is not annotated in the same way?